go.mod

```diff
@@ -10,7 +10,6 @@ require (
 	github.com/Azure/go-autorest/autorest/adal v0.9.5 // indirect
 	github.com/DataDog/datadog-api-client-go v1.0.0-beta.16
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
-	github.com/aws/aws-sdk-go v1.36.21 // indirect
```

Comment:
Now we can remove this too. Feel so good.

Reply:
lol 🤣